### PR TITLE
maintainers: SPEC session

### PR DESCRIPTION
**Pull request by adrinjalali**
To confirm with: Jarrod Millman <jarrod.millman@gmail.com>

### CODE REVIEW

File: content/pages/2022/maintainers.md [hunk 1]
```diff
@@ -22,7 +22,43 @@ a maintainers session.
 All the sessions will take place in the <TBA> room.
 
 ### Schedule
-<TBA>
+
+#### Scientific Python / SPECs
+**Chair: Jarrod Millman**
+**Time: <TBA>**
+
+The Scientific Python project aims to better coordinate the ecosystem and grow
```
Comment (jarrodmillman) [inline content/pages/2022/maintainers.md:30]:
```suggestion
The [Scientific Python](https://scientific-python.org/) project aims to better coordinate the ecosystem and grow
```

File: content/pages/2022/maintainers.md [hunk 2]
```diff
@@ -22,7 +22,43 @@ a maintainers session.
...
+we will discuss project goals and recent technical work.
+
+The Scientific Python project's vision is to help pave the way towards a
+unified, expanded scientific Python community. It focuses its efforts along two
+primary axes: _(i)_ to create a joint community around all scientific projects
+and _(ii)_ to support maintainers by building cross-cutting technical
+infrastructure and tools. In this session we mostly focus on the second aspect.
+
+The project has already launched a process whereby projects can, voluntarily,
+adopt reference guidelines; these are known as SPECs or Scientific Python
```
Comment (jarrodmillman) [inline content/pages/2022/maintainers.md:42]:
```suggestion
adopt reference guidelines; these are known as [SPECs](https://scientific-python.org/specs/) or Scientific Python
```

**Review by jarrodmillman — APPROVED** [review]
LGTM. I thought it may make sense to add a few links so it is easier for folks to read more, but the links aren't necessary.

**Comment by adrinjalali** [comment]
Cool, gonna merge, but feel free to send a PR to change if anything comes up. Or just let me know and I can change them.

**Pull request merged**